torch_tvm/compiler.cpp

```diff
@@ -126,6 +129,7 @@ tvm::relay::Expr TVMCompiler::convertToRelay(
 tvm::relay::Function TVMCompiler::convertToRelay(
     std::shared_ptr<Graph> subgraph,
     TVMContext ctx,
+    torch_tvm::ManagedParamTensors* tvm_param_tensors_ptr,
```

Comment:
can we just use `std::vector<size_t>* immutable_input_indices`?  Adding a new data-structure should be avoided here

Reply:
Why is that? Note that in the current implementation this is needed. We cannot just get away with the indices.